config/settings.py

```diff
@@ -10,75 +10,86 @@
 https://docs.djangoproject.com/en/5.2/ref/settings/
 """
 
+import os
+from dotenv import load_dotenv
 from pathlib import Path
 
+load_dotenv(override=True)
+
 from django.conf.global_settings import STATICFILES_DIRS
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
+CONTACTS_DIR = BASE_DIR / "data"
+CONTACTS_FILE = CONTACTS_DIR / "contacts.csv"
+
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/5.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-!-0=38bo&2e)icpgb2w!9rohcf%16(8ziyr8#q@w6+0so3^^8+'
+SECRET_KEY = os.getenv("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = True if os.getenv("DEBUG") == "True" else False
```

Comment:
Можно проще
```suggestion
DEBUG = os.getenv("DEBUG", "False").lower() == "true"
```
Тогда сможешь задавать и в нижнем и в верхнем регистре слово `false` или `False`